### ISSUE TITLE
Adding a new barcamp results in an Attribute Error. 

### ISSUE THREAD

**Issue by caveman99** [comlounge/camper#162]
calling https://camper.local/b/add results in Attribute Error. Looks like there's info missing about countries?

```
Traceback (most recent call last):
  File "/home/thomas/lib/python2.7/site-packages/starflyer-2.1.1-py2.7.egg/starflyer/app.py", line 481, in __call__
    response = self.handle_exception(request, e)
  File "/home/thomas/lib/python2.7/site-packages/starflyer-2.1.1-py2.7.egg/starflyer/app.py", line 479, in __call__
    response = self.process_request(request)
  File "/home/thomas/lib/python2.7/site-packages/starflyer-2.1.1-py2.7.egg/starflyer/app.py", line 461, in process_request
    response = self.handle_user_exception(request, e)
  File "/home/thomas/lib/python2.7/site-packages/starflyer-2.1.1-py2.7.egg/starflyer/app.py", line 450, in process_request
    response = handler(**request.view_args)
  File "/home/thomas/lib/python2.7/site-packages/starflyer-2.1.1-py2.7.egg/starflyer/handler.py", line 227, in __call__
    rv = getattr(self, method)(**m)
  File "/home/thomas/camper/camper/base.py", line 360, in wrapper
    return method(self, *args, **kwargs)
  File "/home/thomas/camper/camper/barcamps/add.py", line 73, in get
    countries = [(c.alpha2, trans.ugettext(c.name)) for c in pycountry.countries]
  File "/home/thomas/lib/python2.7/site-packages/pycountry-17.9.23-py2.7.egg/pycountry/db.py", line 22, in __getattr__
    raise AttributeError
AttributeError
```

**Comment by mrtopf** [comment]
`pycountry` apparently changed it's attribute names sometime ago..

**Comment by mrtopf** [comment]
fixed in 6913c1454f7fd930d41cc01b9b1195a93a26691b
